### PR TITLE
fix: tweak info text and style

### PR DESCRIPTION
**Pull request by janhenrikoverland**
Change info wording. Use a bullet list for highlighting. Visual result approved by @cooper-joe 

Screenshot from 2020-09-08 17-14-56

### CODE REVIEW

File: src/components/Dashboard/PrintInfo.js [hunk 1]
```diff
@@ -33,9 +21,24 @@ const PrintInfo = ({ isLayout }) => {
                 {`${i18n.t('Print Preview')}: ${infoHeader}`}
             </p>
             <div style={{ maxWidth }}>
-                <p
-                    className={classes.info}
-                >{`${infoPartOne} ${infoPartTwo}`}</p>
+                <p className={classes.info}>
+                    {i18n.t('For best print results:')}
```
Comment (jenniferarnesen) [inline src/components/Dashboard/PrintInfo.js]: Can't have a `:` in the translation string bc it is the namespace separator. I guess just put it outside of the `i18n.t` (you can see that it didn't get added to en.pot)

File: src/components/Dashboard/PrintInfo.js [hunk 2]
```diff
@@ -33,9 +21,24 @@ const PrintInfo = ({ isLayout }) => {
...
+                </p>
+                <p className={classes.info}>
+                    <ul className={classes.infoList}>
+                        <li>{i18n.t('Use Chrome or Edge web browser')}</li>
+                        <li>
+                            {i18n.t(
+                                'Wait for all dashboard items to load before printing this page'
```
Comment (jenniferarnesen) [inline src/components/Dashboard/PrintInfo.js]: Perhaps omit the last two words `this page` since the dashboard can have multiple pages.

**Review by jenniferarnesen — APPROVED** [review]

File: src/components/Dashboard/PrintInfo.js (hunk 2 above)
Comment (janhenrikoverland) [reply src/components/Dashboard/PrintInfo.js]: @cooper-joe

**Comment by jenniferarnesen** [comment]
Approved, just address the comments first.

**Pull request merged**